framework/src/packets/icmp/v6/ndp/options/mod.rs

```diff
@@ -26,6 +27,12 @@ pub enum NdpOption {
     Undefined(u8, u8),
 }
 
+pub trait NdpAddOptions {
```

Comment:
I'd probably just make a generic trait here, e.g. `NdpOptions` for various functions. We usually want to avoid one fn traits. Maybe something more general opens up more reusable functionality?

Reply:
renamed